### ISSUE TITLE
Realms Wiki: Missing Features

### ISSUE THREAD

**Issue by prologic** [scragg0x/realms-wiki#101]
So I recently came across this lovely little wiki engine. It's brilliantly designed (*IHMO*) and love the fact that it's both performance and "just works" (tm) including a nice Docker image!

This issue however is to collect a few thoughts on what I *think* are missing features (*most of which are already separate issues on their own*):

- [ ] [Uploading Media](https://github.com/scragg0x/realms-wiki/issues/68)
- [ ] [Authentiation](https://github.com/scragg0x/realms-wiki/issues/48)
- [ ] [Namespaces/Tags/Categories](https://github.com/scragg0x/realms-wiki/issues/35)
- [ ] [RSS Support](https://github.com/scragg0x/realms-wiki/issues/11)
- [ ] A way to lock down the wiki so it's private (*authenticated only*)

Basically I when I came across this I had this idea of using realms-wiki as a sort-of **private** social publishing platform for family/friends (*not anonymously accessible on the web*). realms-wiki is *close*; but is missing a few key features to make this happen.

The builtin LDAP authentication is great (*not tried it yet*); but this is nice because I was going to tie this in with an LDAP instance to control users/roles.
